easycv/predictors/ocr.py

```diff
@@ -0,0 +1,302 @@
+        dst_img_height, dst_img_width = dst_img.shape[0:2]
+        if dst_img_height * 1.0 / dst_img_width >= 1.5:
+            dst_img = np.rot90(dst_img)
+        return dst_img
+
+    def __call__(self, inputs):
+        # support srt list(str) list(np.array) as input
+        if isinstance(inputs, str):
+            inputs = [inputs]
+        if isinstance(inputs[0], str):
+            inputs = [cv2.imread(path) for path in inputs]
+
+        dt_boxes_batch = self.det_predictor(inputs)
+        boxes_res = []
+        text_res = []
+        for img, dt_boxes in zip(inputs, dt_boxes_batch):
+            dt_boxes = dt_boxes['points']
+            dt_boxes = self.sorted_boxes(dt_boxes)
+            img_crop_list = []
+            for bno in range(len(dt_boxes)):
+                tmp_box = copy.deepcopy(dt_boxes[bno])
+                img_crop = self.get_rotate_crop_image(img, tmp_box)
+                img_crop_list.append(img_crop)
+            if self.use_angle_cls:
+                cls_res = self.cls_predictor(img_crop_list)
+                img_crop_list, cls_res = self.flip_img(cls_res, img_crop_list)
+
+            rec_res = self.rec_predictor(img_crop_list)
+            filter_boxes, filter_rec_res = [], []
+            for box, rec_reuslt in zip(dt_boxes, rec_res):
+                score = rec_reuslt['preds_text'][1]
+                if score >= self.drop_score:
+                    filter_boxes.append(np.float32(box))
+                    filter_rec_res.append(rec_reuslt['preds_text'])
+            boxes_res.append(filter_boxes)
+            text_res.append(filter_rec_res)
+        return boxes_res, text_res
+
+    def flip_img(self, result, img_list, threshold=0.9):
+        output = {'labels': [], 'logits': []}
+        img_list_out = []
+        for img, res in zip(img_list, result):
+            label, logit = res['class'], res['neck']
+            output['labels'].append(label)
+            output['logits'].append(logit[label])
+            if label == 1 and logit[label] > threshold:
+                img = cv2.flip(img, -1)
+            img_list_out.append(img)
+        return img_list_out, output
+
+    def show(self,
+             boxes,
+             rec_res,
+             img,
+             drop_score=0.5,
+             font_path='./doc/simfang.ttf'):
+        img = Image.fromarray(cv2.cvtColor(img, cv2.COLOR_BGR2RGB))
+        txts = [rec_res[i][0] for i in range(len(rec_res))]
+        scores = [rec_res[i][1] for i in range(len(rec_res))]
+
+        draw_img = draw_ocr_box_txt(
+            img,
+            boxes,
+            txts,
+            scores,
+            drop_score=drop_score,
+            font_path=font_path)
+        draw_img = draw_img[..., ::-1]
+        return draw_img
+
+
+def draw_ocr_box_txt(image,
+                     boxes,
+                     txts,
+                     scores=None,
+                     drop_score=0.5,
+                     font_path='./doc/simfang.ttf'):
```

Comment:
font should be added to library resource and use relative package path to load it